src/utils/appeal-utils.js

```diff
@@ -18,6 +18,11 @@ export function transformAppealDataAttributes(appeal) {
       dispute: {
         ...round.dispute,
         lastRoundId: parseInt(round.dispute.lastRoundId, 10),
+        rounds: round.dispute.rounds.map(round => ({
+          ...round,
+          number: parseInt(round.number, 10),
+          jurorsNumber: bigNum(round.jurorsNumber),
```

Comment:
Just wondering, is it possible to reach a number that native JS numbers won't work? E.g. is this value always increasing, even if past jurors deactivate?

I think it is highly unlikely for us to get above 2^53 active jurors.

Reply:
For normal rounds this number won't be too high. It's calculated based on the previous round `jurorsNumber` multiplied by a `appealStepFactor`.
E.g. Current court on mainnet is configured with 3 jurors for first round and an `appealStepFactor` of 3 with a max appeal rounds of 4 so at most it will be 234.

For final rounds the `jurorsNumber` is the number of times the minimum required active balance is held in the registry at the round start term, multiplied  by a final round precision factor of `1000`. 

With current court configurations i think i may be also unlikely for it to get above 2 ^ 53 so we could not convert it to a big number. 
What do you think?

Reply:
Yes, I think it's reasonable to assume we don't need to convert it to a big number then. We will have much bigger problems if we reach 2^53 jurors.